tests/MoBi.Tests/Presentation/SelectSpatialStructureAndMoleculesPresenter.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Collections.Generic;
+using FakeItEasy;
+using MoBi.Core.Domain.Model;
+using MoBi.Core.Domain.Repository;
+using MoBi.Presentation.Presenter;
+using MoBi.Presentation.Views;
+using OSPSuite.BDDHelper;
+using OSPSuite.BDDHelper.Extensions;
+using OSPSuite.Core.Domain.Builder;
+
+namespace MoBi.Presentation
+{
+   public class concern_for_SelectSpatialStructureAndMoleculesPresenter : ContextSpecification<SelectBuildingBlocksForExtendPresenter>
+   {
+      protected IBuildingBlockRepository _buildingBlockRepository;
+      protected ISelectSpatialStructureAndMoleculesView _view;
+
+      protected override void Context()
+      {
+         _view = A.Fake<ISelectSpatialStructureAndMoleculesView>();
+         _buildingBlockRepository = A.Fake<IBuildingBlockRepository>();
+         sut = new SelectBuildingBlocksForExtendPresenter(_view, _buildingBlockRepository);
+      }
+   }
+
+   public class When_Selecting_building_blocks_and_the_view_is_canceled : concern_for_SelectSpatialStructureAndMoleculesPresenter
+   {
+      protected override void Context()
+      {
+         base.Context();
+         A.CallTo(() => _buildingBlockRepository.MoleculeBlockCollection).Returns(new List<MoleculeBuildingBlock> { new MoleculeBuildingBlock() });
```

Comment:
you can just use new [] 
less code, just better

Reply:
Need to teach Copilot that one.